src/library_exceptions.js

```diff
@@ -440,8 +445,26 @@ var LibraryExceptions = {
     catchInfo.free();
     {{{ makeThrow('ptr') }}}
   },
+#endif
 };
 
+
+#if DISABLE_EXCEPTION_CATCHING
+[
+  '__cxa_begin_catch',
+  '__cxa_end_catch',
+  '__resumeException',
+  '__cxa_find_matching_catch',
+  '__cxa_get_exception_ptr',
+].forEach(function(name) {
+  LibraryExceptions[name] = function() { abort(); };
+#if !INCLUDE_FULL_LIBRARY
+  LibraryExceptions[name + '__deps'] = [function() {
+    error('DISABLE_EXCEPTION_CATCHING was set, which means no C++ exception catching support code is linked in, but such support is required by symbol `' + name + '`. Either link with DISABLE_EXCEPTION_CATCHING=0 (if you do want exception catching) or compile all source files with DISABLE_EXCEPTION_CATCHING=1 (the default) (so that no exception catching support code is required).');
```

Comment:
this error is part of another PR I think?

Reply:
Sorry, I'm not the best at managing chains of PRs in github.   This change is designed to land prior to the larger PR where you first saw this.